docs/guide/request_lifecycle.md

```diff
@@ -0,0 +1,253 @@
+
+The `ezpublish.siteaccess_match_listener` service:
+
+- finds the current SiteAccess using the `SiteAccess\Router` (`ezpublish.siteaccess_router`) regarding the [SiteAccess Matching configurations](multisite/siteaccess_matching.md),
+- adds the current SiteAccess to the `Request` object's attribute **`siteaccess`**,
+- then dispatches the `ezpublish.siteaccess` event (`MVCEvents::SITEACCESS`).
+
+The `SiteAccessListener` (`ezpublish.siteaccess_listener`) subscribes to this `ezpublish.siteaccess` event with top priority (priority 255).
+The `SiteAccessListener` adds the **`semanticPathinfo`** attribute, the path without SiteAccess indications ([`URIElement`](multisite/siteaccess_matching/#urielement), [`URIText`](multisite/siteaccess_matching/#uritext),
+or [`Map\URI`](multisite/siteaccess_matching/#mapuri) implementing the `URILexer` interface) to the request.
+
+### Routing
+
+Finally, the `Symfony\Component\HttpKernel\EventListener\RouterListener` (`router_listener`) (priority 32), which also listens to the `kernel.request` event,
+calls `eZ\Publish\Core\MVC\Symfony\Routing\ChainRouter::matchRequest` and adds its returned parameters to the request.
+
+#### `ChainRouter`
+
+The [`ChainRouter`](https://symfony.com/doc/current/cmf/components/routing/chain.html) is a Symfony Content Management Framework (CMF) component. [[= product_name =]] makes it a service named `ezpublish.chain_router`.
+It has a collection of prioritized routers where to find one matching the request.
+The `ChainRouter` router collection is built by the `ChainRoutingPass`, collecting the services tagged `router`.
+The `DefaultRouter` is always added to the collection with top priority (priority 255).
+
+#### `DefaultRouter`
+
+`DefaultRouter` (`router.default`):
+The `DefaultRouter` tries to match the `semanticPathinfo` against routes, close to [the way pure Symfony does](https://symfony.com/doc/current/routing.html), by extending and using `Symfony\Component\Routing\Router`.
+If a route matches, the controller associated to it has the responsibility to build a `View` or `Response` object.
+
+### `UrlWildcardRouter`
+
+`UrlWildcardRouter` (`ezpublish.urlwildcard_router`):
+If [URL Wildcards](url_management/#url-wildcards) have been enabled, then the `URLWildcardRouter` is the next tried router.
+If a wildcard matches, the request's `semanticPathinfo` is updated and the router throws a `ResourceNotFoundException` to continue with the `ChainRouter` collection's next entry.
+
+### `UrlAliasRouter`
+
+`UrlAliasRouter` (`ezpublish.urlalias_router`):
+This router uses the `UrlAliasService` to associate the `semanticPathinfo` to a Location.
+If it finds a Location, the request receives the attributes **`locationId`** and **`contentId`**, **`viewType`** is set to `full`, and the **`_controller`** is set to `ez_content:viewAction` for now.
+
+The `locale_listener` (priority 16) sets the request's **`_locale`** attribute.
+
+!!! caution "Permission control"
```

Comment:
```suggestion
!!! note "Permission control"
```
Actually when I think about it, these don't need to be cautions, just notes (cautions are reserved for dangerous/critical notes)